src/cache/surface_albedo.jl

```diff
@@ -110,7 +110,7 @@ so the initial callback initialization doesn't lead to NaNs in the radiation mod
 Subsequently, the surface albedo will be updated by the coupler.
 """
 function set_surface_albedo!(Y, p, t, ::CouplerAlbedo)
```

Comment:
There's another copy of this function in ClimaCoupler too.

This is entire function a hack and we should remove it. 

This function is here because there's no way to not set the albedo in ClimaAtmos, essentially.